docs/deploying/deploy-nextjs-app.mdx

```diff
@@ -33,6 +35,25 @@ If you omit the `--prod` flag it will deploy it to a preview/test URL.
 
 **Make sure to check the values of your Scaffold Configuration before deploying your NextJS App.**
 
+## Deploy to BG IPFS
+
+You can also deploy your app to BG IPFS by running:
```

Comment:
Maybe we link the [BG IFPS](https://www.bgipfs.com/) to the website? So that people know what's it about?